src/config/dynamic-theme-fixes.config

```diff
@@ -708,6 +708,29 @@ CSS
 
 ================================
 
+google.*
+google.*.*
+
+INVERT
+.gb_hc
+.gb_ec
+.gb_x.gb_Vb
+.gb_x.gb_Ub
+.gb_0b
+#dictionary-modules img
+a.gb_b > div
+.gb_D.gb_oc
+
+CSS
+.RNNXgb {
+    box-shadow: 0 0 2px 0 ${rgba(0,0,0,0.16)}, 0 0 0 1px ${rgba(0,0,0,0.08)} !important;
+}
+.Gor6zc {
+    background-color: white !important;
```

Comment:
Please double check if `${color}` pattern should be used to make it look good with light color scheme.

Reply:
I did not change this part of the code, and I don’t know how it works.

Reply:
OK